tools/shared/src/main/scala/org/scalajs/core/tools/javascript/Printers.scala

```diff
@@ -125,22 +162,28 @@ object Printers {
             printRow(trees, "(", ", ", ")")
```

Comment:
(unfold the lines above this line)
This `case tree @ Block(trees)` should be written as `case tree: Block`. `Block` is not a case class, so its `unapply` is "real".